<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hdf5tab</class>
 <widget class="QWidget" name="hdf5tab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>719</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>File preview:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="hdf5previewtree">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="hdf5previewtable"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" rowspan="2">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Behaviour</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="hdf5helpbutton">
          <property name="toolTip">
           <string>Help on how CSV files should be formatted</string>
          </property>
          <property name="text">
           <string>Help</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Locale</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <widget class="HistoryValueCombo" name="hdf5numfmtcombo">
          <property name="toolTip">
           <string>Numerical format of numbers in file:
 System - what this computer is set to use
 English - format 123,456.78
 European - format 123.456,78</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Dates</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="HistoryCombo" name="hdf5datefmtcombo">
          <property name="toolTip">
           <string>Format for dates and times in file.
This will be combination of
YYYY, YY, MM, M, DD, D, hh, h, mm, m, ss and s
separated by |</string>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Numerics</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistoryCombo</class>
   <extends>QComboBox</extends>
   <header>historycombo.h</header>
  </customwidget>
  <customwidget>
   <class>HistoryValueCombo</class>
   <extends>QComboBox</extends>
   <header>historyvaluecombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
